protocols/v2/noise-sv2/src/signature_message.rs

```diff
@@ -32,6 +32,7 @@ impl SignatureNoiseMessage {
         if self.valid_from <= now && self.not_valid_after >= now {
             let secp = Secp256k1::verification_only();
             let (m, s) = self.split();
+            // m = SHA-256(version || valid_from || not_valid_after || server_public_key)
```

Comment:
`server_static_key` (all three key types are "public": ephemeral, static and authority)